src/SignalR/clients/csharp/Client/test/FunctionalTests/HubConnectionTests.cs

```diff
@@ -1076,7 +1076,7 @@ bool ExpectedErrors(WriteContext writeContext)
             }
         }
 
-        [Theory(Skip = "Will be fixed by https://github.com/dotnet/corefx/issues/36901")]
+        [Theory]
         [MemberData(nameof(HubProtocolsAndTransportsAndHubPaths))]
```

Comment:
This probably only needs to be varied over the different HubProtocols but meh

Reply:
Good call.